backend/sass/common.blocks/_setup.scss

```diff
@@ -280,7 +275,7 @@ $kadena-orange: #F3AB3C;
   }
 }
 
-.setup__fullscreen .setup__message-wrapper > .message {
+.setup__message-wrapper > .message {
```

Comment:
Again, we should avoid the nesting. So this should probably simply be `.setup__message`, without any nesting. Same for the `.setup__checkbox` below.